test/hca_metadata_api/test.py

```diff
@@ -356,15 +358,16 @@ def _test_bundle(self,
                      **assertion_kwargs
                      ) -> Bundle:
 
-        manifest, metadata_files = self._canned_bundle(deployment, uuid, version)
+        manifest, metadata_files, links = self._canned_bundle(deployment, uuid, version)
 
         return self._assert_bundle(uuid=uuid,
                                    version=version,
                                    manifest=manifest,
                                    metadata_files=metadata_files,
+                                   links=links,
                                    **assertion_kwargs)
 
-    def _assert_bundle(self, uuid, version, manifest, metadata_files,
+    def _assert_bundle(self, uuid, version, manifest, metadata_files, links,
```

Comment:
```suggestion
    def _assert_bundle(self, uuid, version, manifest, metadata, links,
```
To match the changes in commit `Rename attribute and derived variables`